store/products.ts

```diff
@@ -22,19 +21,24 @@ const productsModule: Module<ProductsState, any> = {
     }
   },
   actions: {
-    async init({ dispatch, commit }) {
+    init({ dispatch, commit }, columns) {
+      const isMobile = columns <= 3
       const rcs = sessionStorage.getItem('rcs')
       commit('newRcs', rcs)
       commit('clearProducts')
-      const placements = isMobile()
-        ? 'home_page.2020_start_few'
-        : 'home_page.2020_start_full'
+      const action = isMobile ? 'loadFew' : 'loadFull'
```

Comment:
This is not the solution I was thinking about. Because we have a delay for knowing if the client is mobile, we will load "few" first for everyone, and then add the products from "full", if desktop.

Don't you agree? Do you have another solution? #1 mention a few.

Reply:
I don't really follow. This run `loadFew` on mobile and `loadFull` on desktop. I thought this would be a simpler solution that to (on desktop) `loadFew` first and then `loadFull`. It would also give more accurate data

Reply:
I don't want to wait for DOM paint width of container before loading the content, so this is why I picked the solution to load few first for everyone.

Reply:
@coopjakob I understand your concern, see current code for an updated implementation 👍